ci/cron/src/Main.hs

```diff
@@ -240,9 +240,13 @@ fetch_gh_versions pred = do
 
 fetch_s3_versions :: DocOptions -> IO Versions
 fetch_s3_versions opts = do
-    dropdown <- fetch "versions.json" False
+    -- On the first run, this will fail so treat it like an empty file.
+    -- We could technically remove the catch later.
+    dropdown <- fetch "versions.json" False `catchIO`
+      (\_ -> pure [])
```

Comment:
It'd be nice to be more specific about this error. I'd prefer to treat a connection failure or an authorization error differently to "404 not found".

That said, I'm not going to block this fix on that.

Reply:
Fully agree but unfortunately the error comes from shelling out to `aws s3`. I don’t want to start parsing the output of that for probably obvious reasons and the exit code is just `1`. The two nicer solutions here would be either just use an s3 library instead of shelling out or call a command to check if the file exists but either seemed a bit more complex than just a bugfix.